components/DropdownLight/DropdownLight.unit.test.jsx

```diff
@@ -86,11 +86,50 @@ describe('<DropdownLight />', () => {
     expect(input.name).toMatch(INPUT_NAME);
   });
 
-  it('should close Dropdown Options when user press Escape', () => {
-    render(<DropdownLight items={itemsStringMock} />);
+  it('should check if it is disabled', () => {
+    render(<DropdownLight items={itemsStringMock} disabled />);
 
     const dropdown = screen.getByRole('button');
     userEvent.click(dropdown);
+    expect(screen.queryByRole('list')).not.toBeInTheDocument();
+  });
+
+  it('should check if it is required', () => {
+    const { container } = render(
```

Comment:
We no longer use the "container", remember I mentioned that to you? Now we use the "screen"